clusters/app.ci/supplemental-ci-images/coreos/cosa-buildroot.yaml

```diff
@@ -30,13 +31,48 @@ items:
       dockerStrategy:
         from:
           kind: "ImageStreamTag"
-          name: "coreos-assembler:latest"
-        dockerfilePath: Dockerfile.dev
+          name: "fedora:34"
+        dockerfilePath: ci/Dockerfile
+      type: Docker
+    triggers:
+    - type: ConfigChange
+    - imageChange:
+        from:
+          kind: ImageStreamTag
+          name: coreos-assembler:imported-latest
+      type: ImageChange
+
+# Main branch based on 'rawhide'
+# Future branch
```

Comment:
"Future branch" -- are you thinking of having a `rawhide` branch in cosa? I think building for N+1 makes a lot of sense, though I'd rather just keep adapting things in the main branch to work on both N+1 and the current release. We'd need also to make sure something is exercising that image. But we can look at that down the road.